sb3_examples/3dball_num_envs.py

```diff
@@ -0,0 +1,51 @@
+from math import ceil
+
+from baselines.common.schedules import LinearSchedule
+from stable_baselines3 import PPO
+from stable_baselines3.common.vec_env import VecMonitor
+
+from gym_unity.envs import make_mla_sb3_env, LimitedConfig
+
+TOTAL_TAINING_STEPS_GOAL = (
+    500000
+)  # Same as config for CI 3dball... Not sure if MLA steps == SB3 steps.
+NUM_ENVS = 12
+STEPS_PER_UPDATE = 2048
+
+
+# NOTE: This only achieves ~90/100 reward and is just a POC. Needs tuning to be useful.
+def main():
+    env = make_mla_sb3_env(
+        LimitedConfig(
+            env_path="/Users/henry.peteet/Documents/RandomBuilds/3DBallSingleNoVis",
```

Comment:
TODO: Make this generic before merging anything. This is a custom env where I took 3DBall, deleted all but 1 agent and made a server build.